tools/x-workbench/register-components.js

```diff
@@ -1,4 +1,7 @@
+// TODO Need to change the relative path to '@financial-times/x-gift-article/stories'
+// after releasing new version of x-dash with x-gift-article
 const components = [
+	require('../../components/x-gift-article/stories'),
```

Comment:
i'm not sure what you mean?

Reply:
oh, instead of using the npm/lerna linked packages? it could but if we ever move the folders around we'd have to update the paths. with lerna links the actual location of the packages doesn't matter, `lerna bootstrap` will be able to locate and link them no matter what

Reply:
Ah, ok. I'm lerna learner still.